dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SharingController.java

```diff
@@ -389,7 +391,7 @@ public void searchUserGroups( @RequestParam String key, @RequestParam( required
             throw new WebMessageException( WebMessageUtils.conflict( "Search key not specified" ) );
         }
 
-        int max = pageSize != null ? pageSize : Integer.MAX_VALUE;
+        int max = pageSize != null ? pageSize : DEFAULT_SHARING_SEARCH_PAGE_SIZE;
```

Comment:
I think we have `Paging.DEFAULT_PAGE_SIZE` can we use that?

Reply:
Yes agree, let's use that one (which is 50) for consistency.

Reply:
Good, done.